manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "manifest_version": 2,
   "name": "Orbit",
-  "version": "0.3.2",
+  "version": "0.3.3",
```

Comment:
There’s a few more instances where the version number needs bumping—you can look at [this commit](https://github.com/orbit-love/orbit-browser-extension/commit/584a574893294006a3db5dfd0b48b793504c39df) for reference

Reply:
I usually apply the bump after all other commits, then do a git tag and a github release. We can look at this together if you’d like (in addition to pushing the update to the Chrome Web Store)